reliquery/storage.py

```diff
@@ -365,6 +387,18 @@ def get_all_relic_tags(self) -> List[Dict]:
 
         return tags
 
+    def get_all_relic_data(self) -> List[Dict]:
+        relic_types = {path.split("/")[0] for path in self.list_key_paths([""])}
```

Comment:
ok i see thanks I didn't think of that. In the case of s3 storage I am splitting because list keys returns a list of path like strings of all contents of the bucket from the key given.

`"/relics/something/anotherthing"`
If the path seprator can be anything then perhaps I should rethink this process of creating relic data.